### ISSUE TITLE
Index mismatch in userOption Yaxis and Yaxis array

### ISSUE THREAD

**Issue by sangeethjohn01** [highcharts/highcharts#13427]
#### Expected behavior
According to the live demo I shared below,
Step 1. While clicking on the add series button it will create two multiple y series data
Step 2. Then click on the remove button It will remove the first axis and corresponding data.
Step 3. click on the export button it is supposed to produce the image but it's not, instead of showing image it's throwing an error in the console.
>>Highcharts error #18

--Observation--
In user option, Y-axis data array and Y-axis data array length is different(which is because we enabled the navigator) and while deleting the specific id it's deleting the last element from user option Y-axis data array  

#### Actual behaviour
It should show the image like producing while clicking export for the first time after step 1.

#### Live demo with steps to reproduce
http://jsfiddle.net/s8jzc26h/1/

**Comment by sebastianbochan** [comment]
Hi @sangeethjohn01,
Thank you for reporting about the problem. The issue is known and has been already reported here: https://github.com/highcharts/highcharts/issues/13020.

Im closing the ticket as duplicate of https://github.com/highcharts/highcharts/issues/13020.
